tests/installation/disk_activation.pm

```diff
@@ -23,24 +23,35 @@ sub format_dasd {
     }
 }
 
+sub add_zfcp_disk {
+    my $channel = shift;
+    assert_screen 'zfcp-disk-management';
+    send_key 'alt-a';
+    assert_screen 'zfcp-add-device';
+    send_key 'alt-a';
+    assert_screen 'zfcp-channel-select';
+    for (1 .. 8) { send_key "backspace"; }
```

Comment:
What's about pressing ctrl-a + del to clean the field? I would also wrap it in wait_screen_change, so we don't start typing before removed the content.

Reply:
from experience on z/VM key combinations with key modifiers might be not 100% reliable...
I'd go with this one for now

Reply:
Should we not try to understand what key presses go missing over VNC?